themes/src/themes/parent-theme/collections/table.overrides

```diff
@@ -8,7 +8,9 @@
   thead {
     background: #1d2630;
     color: @white;
-    
+    background: none;
```

Comment:
Is there a reason why we're removing the background colour here and adding it to the hover state?